### DISCUSSION TITLE
Proposing research / reading and writing group

### DISCUSSION THREAD

**Discussion by Liyubov** [life-itself/community#350] [Research]
As discussed in the last meeting, we wanted to do more general reading/writing group on subjects, which are interesting and important.
Maybe this can be yet another format for our Friday meetings. 
@rufuspollock @msantolini 
We started to discuss about Chambers 1994 paper on Rural Participatory Appraisal. 

John and Simon also nomitated some papers, feel free to post them here in the proposition.
Thank you!

**Comment by baouroux** [comment]
A fabulous paper on Rural Participatory Appraisal, thanks for sharing Liubov...

Herewith a reference to the paper I mentioned on bringing together the cognitive "commons" (a term I'm taking from Simon Grant in our last call!), on the developmental and cognitive lens applied to how democratic dialogue unfolds:

- [Rethinking Democratic Deliberation: The Limits and Potential of Citizen Participation](https://www.researchgate.net/publication/248876356_Rethinking_Democratic_Deliberation_The_Limits_and_Potential_of_Citizen_Participation)

July 2007
DOI:[10.1057/palgrave.polity.2300073](http://dx.doi.org/10.1057/palgrave.polity.2300073)
Shawn-Rosenberg - University of California, Irvine 

It is from these kinds of papers, that I am drawn to the subject of how we coordinate the Good, True and Beautiful in all our "commons", where the above paper I feel fits into the absolutely necessary compenent of the "rational" True, but needs to be held at the same time within the context of the Good and the Beautiful. 

Of course, there are immediate dangers if ever the rational overrides the others, and vice-versa. Everything is a balance and a dynamic. How might we navigate these dynamics more consciously?!
